app/scripts/modules/core/src/managed/ObjectRow.module.css

```diff
@@ -40,8 +38,6 @@
   text-overflow: ellipsis;
   white-space: nowrap;
   margin-right: 16px;
-
-  pointer-events: none;
```

Comment:
@alanmquach I deleted this rule since it was preventing the link from being clicked. I didn't quite understand your related comment, so want to check if I'm introducing any issue here.

Reply:
kinda looks to me like it was left over from some implementation details of Greg's prototype code